tests/test_equilibrium.py

```diff
@@ -464,3 +465,143 @@ def test_backward_compatible_load_and_resolve():
     f_obj = ForceBalance(eq=eq)
     obj = ObjectiveFunction(f_obj)
     eq.solve(maxiter=1, objective=obj)
+
+
+@pytest.mark.unit
+def test_shifted_circle_geometry():
+    """
+    In this test, we calculate a low-beta shifted circle equilibrium with DESC.
+
+    We then compare the various geometric coefficients with their respective analytical
```

Comment:
is there a reference for where the analytical forms come from?

Reply:
There are a couple of references here and there but each one has slight variations. The point here is not to obtain the exact analytical expression with the right factors but to have something to compare with. 

For this case, we'll do the bounce-averaged drifts and the numerical solutions will be a function of the pitch angle whereas the analytical values will be of the form analytical_geometric_coefficients * Elliptic_integrals(function of the pitch angle) (from scipy).

Reply:
that's fine, I just wanted a reference for where the formulas implemented below come from.

Reply:
Ok, I have added a reference